http/util/src/main/java/org/wildfly/security/http/util/SetMechanismInformationMechanismFactory.java

```diff
@@ -73,6 +74,7 @@ public String getMechanismName() {
             public void evaluateRequest(HttpServerRequest request) throws HttpAuthenticationException {
                 String host = request.getFirstRequestHeaderValue(HOST);
                 String resolvedHostName = null;
+                URI requestedUri = request.getRequestURI();
```

Comment:
Is this needed here? It doesn't look like it's used.

Reply:
Removed